docs/zh_CN/TrainingService/AMLMode.rst

```diff
@@ -0,0 +1,128 @@
+**在 Azure Machine Learning 上运行 Experiment**
+===================================================
+
+NNI 支持在 `AML <https://azure.microsoft.com/en-us/services/machine-learning/>`__ 上运行 Experiment，称为 aml 模式。
```

Comment:
most azure documents have Chinese version, we can provide them.

Reply:
updated all.